src/workflows/tests/vervet-resolver.test.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as os from "os";
+import { loadVervetResoucePaths } from "../vervet-resolver";
+
+describe("vervet init", async () => {
+  const originalwd = process.cwd();
+  var tmpDir: string = "";
```

Comment:
```suggestion
  let tmpDir: string = "";
```

Reply:
Good call. TIL... https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var